README.md

```diff
@@ -1,3 +1,4 @@
+[![Stories in Ready](https://badge.waffle.io/shuttleofx/ShuttleOFX.png?label=ready&title=Ready)](https://waffle.io/shuttleofx/ShuttleOFX)
```

Comment:
Better to put that a bit later in the readme (not before the title).
See how it looks: https://github.com/shuttleofx/ShuttleOFX/blob/73b4d47193ea5400b6fe191d14967b551960efeb/README.md